Apps/Immunization/test/unit/Services.Test/ImmunizationServiceTests.cs

```diff
@@ -627,69 +284,15 @@ private static RequestResult<PhsaResult<ImmunizationResponse>> GetPhsaResult(Imm
                 {
                     LoadState = new() { RefreshInProgress = false },
                     Result = new(
-                        new List<ImmunizationViewResponse>(),
-                        new List<ImmunizationRecommendationResponse>
-                        {
-                            immzRecommendationResponse,
-                        }),
+                        [],
+                        [immzRecommendationResponse]),
```

Comment:
Nit: now that these lines are much shorter, they can probably all fit on a single line.

Reply:
Need to configure Rider.  Change next time.